### PR TITLE
Small nits and cleanup of proposal life cycle.

### PR DESCRIPTION
**Pull request by heyawhite**
No new information, just cleaning up / clarifying some language. No urgency.

### CODE REVIEW

File: site/en/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 1]
```diff
@@ -13,50 +13,48 @@ authors:
...
 ## From proposal to web standard
 
-There is a large amount of ecosystem input shaping this work, at every stage of
-development. This process may be familiar to web developers, but may be new to
-other industry stakeholders who will use these purpose-built APIs&mdash;and
-whose expertise is critical to this initiative.
+At every stage of development, the ecosystem gives critical feedback which
+shapes the Privacy Sandbox. This process may be familiar to web developers, but
+may be new to other industry stakeholders who will use these purpose-built
+APIs&mdash;and whose expertise is critical to this initiative.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/proposal-lifecycle/index.md:47]: delete the em dash? @heyawhite

File: site/en/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 2]
```diff
@@ -13,50 +13,48 @@ authors:
...
 Chromium (the open source platform behind many modern browsers) has written
-about the [feature development
-process](https://www.chromium.org/blink/launching-features/) for all
-technologies which aim to become a web standard. Because of the critical nature
-of privacy and security on the web, we expect and encourage large amounts of
-discussion and feedback before testing begins.
+about the [feature development process](https://www.chromium.org/blink/launching-features/)
+for all technologies which aim to become a web standard. Because of the
+critical nature of privacy and security on the web, we expect and encourage
+large amounts of discussion and feedback before testing begins.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/proposal-lifecycle/index.md:40]: suggestion: a great deal of vs large amounts of

File: site/en/docs/privacy-sandbox/proposal-lifecycle/index.md
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/proposal-lifecycle/index.md:1]: The reduced User-Agent --- line 203 does that need "string" after it?

**Comment by nancymic2** [comment]
> No new information, just cleaning up / clarifying some language.

sorry left a few nits as well @heyawhite

File: site/en/docs/privacy-sandbox/proposal-lifecycle/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/proposal-lifecycle/index.md:47]: Maybe I'd better suggest these changes later and just merge this now

**Review by nancymic2 — APPROVED** [review]

**Review by nancymic2 — APPROVED** [review]

**Comment by heyawhite** [comment]
No publishing urgency, so @nancymic2 if you have additional nits, by all means you can suggest.

**Pull request merged**